### PR TITLE
Add the JAX logo to sphinx docs

### PR DESCRIPTION
**Pull request by shoyer**
Here's a preview:

image

The background color is easy to adjust.

### CODE REVIEW

File: docs/_static/style.css [hunk 1]
```diff
@@ -0,0 +1,5 @@
+@import url("theme.css");
+
+.wy-side-nav-search {
+  background-color: #fcf;
```
Comment (mattjj) [inline docs/_static/style.css]: Hmm maybe just #fff?

Comment (mattjj) [reply docs/_static/style.css]: (Or whatever is inherited.)

Comment (shoyer) [reply docs/_static/style.css]: done

Comment (shoyer) [reply docs/_static/style.css]: the default color is very blue, not good contrast:

image

**Comment by mattjj** [comment]
Thanks! Now I just want to fiddle with the color all day...

**Pull request merged**